openid-4-verifiable-presentations-1_0.md

```diff
@@ -2454,57 +2281,46 @@ AnonCreds is a Credential format defined as part of the Hyperledger Indy project
 
 To be able to request AnonCreds, there needs to be a set of identifiers for Verifiable Credentials, Verifiable Presentations ("proofs" in Indy terminology) and crypto schemes.
 
-Credential format identifier is `ac_vc` for a Credential, and `ac_vp` for a Presentation.
-
-Identifier for a CL-signature crypto scheme used in the examples in this section is `CLSignature2019`.
-
-### Example Credential
-
-The following is a non-normative example of an AnonCred Credential that will be used throughout this section. 
+The Credential format identifier is `ac_vp` to request a Presentation.
 
-<{{examples/credentials/ac_vc.json}}
+The identifier for the CL-signature crypto scheme used in the examples in this section is `CLSignature2019`.
 
-The most important parts for the purpose of this section are `scheme_id` parameter and `values` parameter that contains the actual End-User claims. 
+### DCQL
 
-### Presentation Request 
+#### Parameters in the `meta` parameter in Credential Query {#anoncreds_meta_parameter}
 
-#### Request Example {#anoncreds_request}
+The following are AnonCreds specific parameters in the `meta` parameter in a Credential Query as defined in (#credential_query):
 
-The following is a non-normative example of an Authorization Request:
+`schema_id_values`:
+: OPTIONAL. An array of strings that specifies the allowed values for the `schema_id` of the requested Verifiable Credential. It MUST be a valid scheme identifier as defined in [@Hyperledger.Indy].
 
-<{{examples/request/request.txt}}
+`cred_def_id_values`:
+: OPTIONAL. An array of strings that specifies the allowed values for the `cred_def_id` of the requested Verifiable Credential. It MUST be a valid credential definition identifier as defined in [@Hyperledger.Indy].
 
-The following is a non-normative example of the content of the `presentation_definition` parameter:
+`proof_type_values`:
+: OPTIONAL. An array of strings that specifies the types of proofs that the Verifier accepts to be used in the Verifiable Presentation, for example `CLSignature2019`.
 
-<{{examples/request/pd_ac_vc.json}}
+#### Claims Matching
 
-The `format` object in the `input_descrioptor` element uses the format identifier `ac_vc` as defined above and sets the `proof_type` to `CLSignature2019` to denote this descriptor requires a Credential in AnonCreds format signed with a CL signature (Camenisch-Lysyanskaya signature). The rest of the expressions operate on the AnonCreds JSON structure.
+When used in the context of AnonCreds, the `claims_path` parameter always matches on the contents of the `values` key in the JSON-representation of the Verifiable Credential.
```

Comment:
This is a change of how the PEX definition handled Anoncreds credentials. The matching would be done on the outer wrapper (so not within the `values`), which would allow you to also query for e.g. the `issuer_did` / `issuer_id`. 

With this change you can only use DCQL to query on the values, and the meta defines `schema_id_values`, and `cred_def_id_values`. 

So I think we should either add a meta value to filter by did, allow dids to be used as trusted authorities, or query based on the outer object (so `['values', 'name']`), to not break issued based matching for anoncreds credentials

Reply:
Thanks, I'll change it to match on the outer object then.

Reply:
Looking at the AnonCreds specification again and there is no `issuer_did` 
 property on the credential. It is allowed as a query param in an AnonCreds proof request restriction. 

So the change I proposed is not sufficient to query by issuer did. I've opened #494 to support `did` as trusted authorities.

Reply:
However with `schema_id_values` and `cred_def_id` values already being part of the meta, i think querying directly on the values would be okay then, as it does not solve the issuer did querying. My apologies for giving a wrong solution.

Reply:
I'll change it back to matching on values then :-)